packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -3,14 +3,20 @@
  * Licensed under the MIT License.
  */
 
-import { strict as assert } from "assert";
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
The number of violations to these `any`-related rules in this module was well beyond what I was willing to tackle as a part of this PR. Since this is test code, I'm just ignoring these for now.